### PR TITLE
Update UserGuide.md

### LINKED ISSUE #215: [PE-D][Tester D] Command summary for list does not match the actual command
The command for listing all appointment is `list appts`, not `list appointments` as seen in the command summary.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jetlfj/ped#6

### PR DESCRIPTION
**Pull request by JovonLim**
Closes #215 #210 #204 #200

### CODE REVIEW

**Review by NicholasTYD — APPROVED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -377,10 +380,10 @@ Examples:
 
 Shows the most recent 10 commands that was inputted.
 Allows cycling through those commands to reduce the need of retyping similar commands fully.
+The history is looped, meaning that when you reach the end, the next cycle will bring it back to the start.
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:** Every new valid command will reset the index back to the top of the history. 
+**Tip:** Every new valid command will reset the index back to the top of the history.
 Click Down Arrow to get the most recent command immediately and Up Arrow to get the last command in the history.
-</div>
```
Comment (pwjj2000) [inline docs/UserGuide.md:383]: Actually why are we removing the :bulb: div? Is it not fine for it to remain?

**Review by pwjj2000 — COMMENTED** [review]
Need some clarification on the removal of :bulb: div first since I don't think it's necessary to remove

File: docs/UserGuide.md (hunk 1 above)
Comment (JovonLim) [reply docs/UserGuide.md:383]: Oops i forgot to edit back because initially wanted to put the loop part together but decided against it

**Review by pwjj2000 — APPROVED** [review]
LGTM!

**Pull request merged**